ISSUE_TEMPLATE

```diff
@@ -0,0 +1,16 @@
+#### Expected behavior
+
+#### Actual behavior
+
+#### Steps to reproduce the behavior
+
+---
+##### Link to PDF file
```

Comment:
Perhaps it would be a good idea to also indicate that GitHub now supports attaching PDF files to issues?